src/DssDirectDepositAaveDai.sol

```diff
@@ -202,10 +207,13 @@ contract DssDirectDepositAaveDai {
     function _rdiv(uint256 x, uint256 y) internal pure returns (uint256 z) {
         z = _mul(x, RAY) / y;
     }
+    function min(uint256 x, uint256 y) internal pure returns (uint256 z) {
+        z = x <= y ? x : y;
+    }
 
     // --- Administration ---
     function file(bytes32 what, uint256 data) external auth {
-        require(live == 1, "DssDirectDepositAaveDai/not-live");
+        require(vat.live() == 1, "DssDirectDepositAaveDai/no-file-during-shutdown");
```

Comment:
This check should not replace the original `live` check. It should be in addition.

Reply:
Hmm what is the intention to block that?
I mean if the module is caged, and governance decides to change the `bar` nothing happens as still goes through the other paths.

Reply:
Yeah that's true, we could actually remove the check at all.

Reply:
The check was removed from the file that sets the `bar`.